cypress/e2e/agreements/agreement-line-create-multiple.cy.js

```diff
@@ -0,0 +1,224 @@
+    }).then((packages) => {
+      if (packages?.length === 0) {
+        LocalKBAdminAppInteractor.openLocalKbAdminApp();
+        LocalKBAdminAppInteractor.uploadJsonFileAndAwaitCompletion(fileName);
+      }
+    });
+    AgreementAppInteractor.openAgreementsApp();
+    AgreementAppInteractor.createAgreement(agreement);
+    AgreementAppInteractor.openLocalKB();
+    AgreementAppInteractor.searchForPackage(packageName);
+    PackageViewInteractor.addToBasket();
+    cy.logout();
+  });
+
+  after(() => {
+    console.log('delete users');
+    cy.getAdminToken();
+    cy.deleteUserViaApi(editUser.userId);
+    cy.deleteUserViaApi(viewUser.userId);
+
+    console.log('delete agreement line and agreement');
+    cy.login(Cypress.env('login_username'), Cypress.env('login_password'));
+    cy.visit('/erm/agreements');
+    AgreementAppInteractor.searchAgreement(agreementName);
+    AgreementViewInteractor.paneExists(agreementName);
+    deleteFirstAgreementLine();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(6000);
+    deleteFirstAgreementLine();
+    AgreementViewInteractor.delete(agreementName);
+    cy.logout();
+  });
+
+  describe('user actions', () => {
+    describe('editUser actions', () => {
+      before(() => {
+        cy.login(editUser.username, editUser.password);
+        cy.getToken(editUser.username, editUser.password);
+      });
+
+      after(() => {
+        cy.logout();
+      });
+
+      it('should open the agreements app', () => {
+        AgreementAppInteractor.openAgreementsApp();
+      });
+
+      it('select agreement from results list', () => {
+        AgreementAppInteractor.searchAgreement(agreementName);
+        AgreementViewInteractor.paneExists(agreementName);
+      });
+
+      it('expand agreement lines accordion in agreement view', () => {
+        AgreementViewInteractor.openAgreementLinesAccordion();
+      });
+
+      it('open agreement line accordion actions and click new agreement', () => {
+        AgreementViewInteractor.newAgreementLine();
+      });
+
+      it('select e-resource from selection', () => {
+        AgreementLineFormInteractor.selectEresource(packageName);
+      });
+
+      it('link e-resource from selection', () => {
+        AgreementLineFormInteractor.linkSelectedEresource();
+      });
+
+      it('check "create another" option', () => {
+        AgreementLineFormInteractor.checkCreateAnother(true);
+      });
+
+      it('save but dont close agreement line', () => {
+        AgreementLineFormInteractor.save();
+        AgreementLineFormInteractor.unlinkSelectedEresource();
+      });
+
+      it('click description field then note to check validation', () => {
+        AgreementLineFormInteractor.checkDescriptionValidation();
+      });
+
+      it('fill description field and re-check validation', () => {
+        AgreementLineFormInteractor.fill({ description });
+        AgreementLineFormInteractor.checkDescriptionValidation();
```

Comment:
In this case you would expect the validation to pass, right? so you can't use the same interactor, you'd probably need 2 different ones, `checkDescriptionIsValid` and `checkDescriptionIsNotValid` 
The reason why it doesn't fail atm is because the function isn't working at all. See my comment below